persistence/jdbc/src/test/java/org/infinispan/persistence/jdbc/stringbased/JdbcStringBasedStoreBatchTest.java

```diff
@@ -0,0 +1,94 @@
+package org.infinispan.persistence.jdbc.stringbased;
+
+import static org.testng.AssertJUnit.assertEquals;
+import static org.testng.AssertJUnit.assertNull;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+import org.infinispan.commons.marshall.StreamingMarshaller;
+import org.infinispan.commons.util.ReflectionUtil;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.marshall.TestObjectStreamMarshaller;
+import org.infinispan.marshall.core.MarshalledEntry;
+import org.infinispan.marshall.core.MarshalledEntryImpl;
+import org.infinispan.persistence.jdbc.configuration.JdbcStringBasedStoreConfigurationBuilder;
+import org.infinispan.persistence.jdbc.connectionfactory.ConnectionFactory;
+import org.infinispan.persistence.jdbc.table.management.TableManager;
+import org.infinispan.persistence.jdbc.table.management.TableName;
+import org.infinispan.persistence.spi.AdvancedLoadWriteStore;
+import org.infinispan.persistence.spi.PersistenceException;
+import org.infinispan.test.AbstractInfinispanTest;
+import org.infinispan.test.fwk.TestCacheManagerFactory;
+import org.infinispan.test.fwk.UnitTestDatabaseManager;
+import org.infinispan.util.PersistenceMockUtil;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+/**
+ * @author Ryan Emerson
+ * @since 9.1
+ */
+@Test(groups = "functional", testName = "persistence.jdbc.stringbased.JdbcStringBasedStoreBatchTest")
+public class JdbcStringBasedStoreBatchTest extends AbstractInfinispanTest {
```

Comment:
Hrmm wouldn't this now be covered by `JdbcStringBasedStoreTest` since you added a similar method to the BaseStoreTest ?